draft-ietf-tls-tls13.md

```diff
@@ -3951,6 +3949,25 @@ by IANA
   "Recommended": ecdsa_secp256r1_sha256, ecdsa_secp384r1_sha384,
   rsa_pss_sha256, rsa_pss_sha384, rsa_pss_sha512, ed25519.
 
+-  TLS ProtocolVersion Registry: This registry formalizes the definition
+  of ProtocolVersion values used to identify TLS versions in TLS messages.
+  Values with the first byte in the range 0-254 (decimal) are assigned
+  via Standards Action {{RFC2434}}. Values with the first byte 255
+  (decimal) are reserved for Private Use {{RFC2434}}. Values with the
+  first byte in the range 0-2 SHALL be marked as reserved for SSL 2.0,
```

Comment:
You can't reserve 0x0-0x2ff here when you just assigned it above.  I would just mark 0x0-0x2ff out of bounds; 0x305-0xfeff as Standards Action and the remainder as private use.